packages/core/test/signal.test.tsx

```diff
@@ -1449,12 +1449,12 @@ describe("computed()", () => {
 		});
 
 		it("should only subscribe to signals listened to", () => {
-			// Here both "B" and "C" are active in the beginnning, but
+			// Here both "B" and "C" are active in the beginning, but
 			// "B" becomes inactive later. At that point it should
 			// not receive any updates anymore.
 			//    *A
 			//   /   \
-			// *B     D <- we don't listen to C
+			// *B     D <- we don't listen to D
```

Comment:
I think this meant that `D` doesn't listen to changes in `C` because ..

> Here both "B" and "C" are active in the beginning

I think this means "active" as the effect is listening to `C` (so `B` is a target in `A` and `C` is a target in `B`) but then it calls `unsub` to unsubscribe which makes `C` contain to targets, so it unsubscribes from `B` and in the same way, `B` will have no targets, so it'll unsubscribe from `A`.

Then setting the value of a `a.value = "aa"` is testing that `B` and `C` compute functions aren't called but `D` must update (`D` doesn't listen to neither `C` nor `B`)

Reply:
Ah okay, I reverted that change.

I thought that part of the diagram was just left over from it being copied from the test above it and the author forgot to update that part of it. It still seems strangely worded to me. I usually think of listening as only going up the graph. Something like "C no longer listens to A, but D does" would make more sense to me. But I'm okay with just leaving it as is if you think how it is currently written was intentionally by the author.